sdk/anomalydetector/ai-anomaly-detector-rest/samples-dev/batchDetectAnomalySample.ts

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+// Copyright (c) Microsoft Corporation.
```

Comment:
Please double check all your samples to make sure only one license header is included